src/screens/AppSettings/PrivacyAndDisplay.tsx

```diff
@@ -318,27 +318,29 @@ function PrivacyAndDisplay({ route }) {
       if (available) {
         if (loginMethod === LoginMethod.PIN) {
           const { keysExist } = await RNBiometrics.biometricKeysExist();
-          if (keysExist) {
+          if (!keysExist) {
             await RNBiometrics.createKeys();
```

Comment:
Why call create keys both here and in 328?

Not familiar with RNBiometrics but shouldn't the code be if keysExist call deleteKeys here? Otherwise we just calll twice the create here